examples/API_v2/lookup_tweets_using_tweet_ids.py

```diff
@@ -0,0 +1,19 @@
+import tweepy
+
+
+# Replace bearer token value with your own
+bearer_token = ""
+
+# Initializing the Tweepy client
+client = tweepy.Client(bearer_token)
+
+# Replace Tweet IDs
+ids = ['1409935014725177344', '1409931481552543749', '1441054496931541004']
+
+# By default the Tweet ID and Tweet text are returned. If you want additional fields, use the appropriate
+# fields and expansions
+tweets = client.get_tweets(ids=ids, tweet_fields=['context_annotations','created_at','geo'])
```

Comment:
`ids` can be a positional argument here.

The example should probably make use of the Tweet fields requested here.